pom.xml

```diff
@@ -49,7 +49,33 @@
                 <artifactId>target-platform-configuration</artifactId>
                 <version>${tycho.version}</version>
                 <configuration>
-                    <resolver>p2</resolver>
+                  <environments>
```

Comment:
As noted in the commit message, this eliminates the "[WARNING] No explicit target runtime environment configuration. Build is platform dependent" warnings seen at the start of every build.